src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -3944,6 +3950,174 @@ namespace HVACVariableRefrigerantFlow {
                 }
             }
 
+            // supplemental heating coil
+            if (!lAlphaFieldBlanks(17) && !lAlphaFieldBlanks(18)) {
```

Comment:
The protection is good here but what if a users forgets to enter one or the other input? Would we want to warn if one or the other were blank but not both?

Reply:
I will look into it if it is better to provide a separate warning.

Reply:
if only one of the two is blank, then I will provide warning message.